src/components/libraryGeneration/SlotCopyComponent.tsx

```diff
@@ -381,6 +382,22 @@ function SlotCopyComponent({
     cleanedOutInputAddresses.set(source.labware.id, source.cleanedOutAddresses ?? []);
   });
 
+  const updateDestinationModeWhenPaginating = () => {
+    //Before selecting any destination
+    if (!destinationSelectionMode.current) {
+      return undefined;
+    }
+    if (destinationSelectionMode.current === DestinationSelectionMode.SCAN) {
+      return destinationSelectionMode.current;
+    }
+    if (selectedDestination.labwareType.name === '96 well plate') {
+      destinationSelectionMode.current = DestinationSelectionMode.PLATE;
```

Comment:
Why does `destinationSelectionMode.current` need to be set inside the function? Isn't the return value of this function sufficient?

Reply:
destinationSelectionMode is a reference to the destination mode check list, here we are updating the destination mode accordingly to the labware type displayed.